bukuserver/README.md

```diff
@@ -149,8 +149,23 @@ Note: the value for BUKUSERVER_REVERSE_PROXY_PATH
 is recommended to include preceding slash and not have trailing slash
 (i.e. use `/foo` not `/foo/`)
 
+Note: At Bukuserver, we have disabled favicon as a default setting in order to prevent any non-user triggered network activity.
+
+Our favicon is generated with the assistance of Google.
+
+It is important to be aware that favicon has the potential to be used for browser fingerprinting,
+a technique used to identify and track a person's web browsing habits.
+
+- Github repo example https://github.com/jonasstrehle/supercookie
+- Paper by Scientists at University of Illinois, Chicago https://www.cs.uic.edu/~polakis/papers/solomos-ndss21.pdf
+- Article https://heise.de/-5027814
```

Comment:
Rather than placing hyperlinks after from their titles, it's better to make these titles into hyperlinks; i.e.
```markdown
- [Github repo example](https://github.com/jonasstrehle/supercookie)
```

Additionally, there's two issues with the last item: it's described too vaguely, and is not in English (unlike everything else here). Thus, it's better to elaborate and link to a translated version as well; e.g.
```markdown
- [Article published in 2021 at Heise Online](https://heise.de/-5027814)
  ([English translation](https://www-heise-de.translate.goog/news/Browser-Fingerprinting-Favicons-als-Super-Cookies-5027814.html?_x_tr_sl=de&_x_tr_tl=en&_x_tr_hl=en))
```